src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -278,6 +278,75 @@ public function networkDisconnect(string $container, string $network, bool $forc
         return $result['code'] == 200;
     }
 
+    /**
+     * Get usage stats of containers
+     * 
+     * @param string $container
+     * @param array<string, string> $filters
+     * 
+     * @return array
+     */
+    public function getStats(string $container = null, array $filters = []): array
+    {
+        // List ahead of time, since API does not allow listing all usage stats
+        $containerIds = [];
+
+        if($container === null) {
+            $filtersSorted = [];
+
+            foreach($filters as $key => $value) {
+                $filtersSorted[$key] = [$value];
+            }
+    
+            $body = [
+                'all' => true,
+                'filters' => empty($filtersSorted) ? new stdClass() : json_encode($filtersSorted)
+            ];
+
+            $containers = \json_decode($this->call('http://localhost/containers/json?'.\http_build_query($body), 'GET')['response'], true);
+            $containerIds = \array_map(fn($c) => $c['Id'], $containers);
```

Comment:
Why aren't we using the `list` method of this adapter here ?
https://github.com/utopia-php/orchestration/blob/114046b9010ccb331deb7e23ef4aff22f903b890/src/Orchestration/Adapter/DockerAPI.php#L409

Reply:
Used `list` method instead.